index.njk

```diff
@@ -2,19 +2,54 @@
 layout: 'layouts/base.njk'
 ---
 
-<h1>
-  <span class="eyebrow">Design Sprint Guide</span>
-  Design, prototype, and test ideas quickly.
-</h1>
+<main class="tbds-app-frame" "tbds-block-stack" "tbds-block-stack--bordered">
+  
+  <header class="tbds-app-frame__header  tbds-margin-block-6">
+    <section class="tbds-inline-stack tbds-inline-stack--gap-6 tbds-inline-size-80%">
+      <div class="tbds-inline-stack__item">
+        <h5 class="tbds-brand-red">Design Sprint Guide <br/>
+            by <a class="#" href="#">thoughtbot</a></h5>
+      </div>
 
-<p>A Product Design Sprint is a technique to quickly design, prototype, and test the viability of an idea, product, or feature. The design sprint consists of 5 phases (typically days), starting with design thinking and ending with a user-tested prototype.</p>
+      <div class="tbds-inline-stack__item tbds-inline-stack__item--push-start tbds-inline-stack--gap-6 tbds-font-style-normal">
+       <span class="">
+        <a href="#">Schedules</a>
+        <a href="#">Exercises</a>
+        <a href="#">FAQs</a>
+        <a href="#">Resources and Glossary</a>
+        </span>
+      </div>
+    </section>
+  </header>
 
-<h2>Why a Product Design Sprint?</h2>
 
-<p>
-  Product Design Sprints orient the team and aims our efforts toward a mutual goal and improves the chances of making something people want.
-</p>
+<body class="tbds-app-frame__body">
 
-<p>
-  Sprints are useful when kicking off a new business, product, or feature, as well as solving problems with an existing product.
-</p>
+    <h1 class="tbds-padding-inline-auto">
+    <span class="eyebrow">Design Sprint Guide</span>
+    <span class="tbds-font-weight-bold">Design, prototype, and <br/> test ideas quickly.</span></h1>
+
+  <section class="tbds-inline-stack tbds-inline-stack--gap-6 tbds-inline-size-60%">  
```

Comment:
Be careful to of indentation here, it is indented the same as the parent, despite being a child. Consistent indentation really helps with readability, perhaps research how to get your code editor to do this for you?